exercises/etl/canonical-data.json

```diff
@@ -0,0 +1,62 @@
+{
+  "transform": {
+    "description": "transforms the input map from a map of value: [ keys ] to a map of key: value for each of the keys",
```

Comment:
If there is the idea that this file could be used to generate a test suite, wouldn't you kind of need the method name to know how to invoke the method in the test call?